src/cargo/core/compiler/context/compilation_files.rs

```diff
@@ -266,6 +266,7 @@ impl<'a, 'cfg: 'a> CompilationFiles<'a, 'cfg> {
         } else {
             ""
         };
+
```

Comment:
Please try to avoid stray newlines like this.

Reply:
Sure